### DISCUSSION TITLE
Difference between licensing terms of code-server vs. VS Code Server

### DISCUSSION THREAD

**Discussion by kumaraish** [coder/code-server#6256] [General] [answered]
I am trying to understand the licensing limitations if I want to bundle code-server with my cloud application.

As per the [docs][1], Out of the box, `code-server` is simply **VS Code in the browser**

Since `code-server` uses open source code of `VS Code`, how does the licensing terms of `VS Code Server` affect usage of `code-server` in a commercial application? As per `VS Code Server` [FAQ][2] and [licensing terms][3] -
 - hosting `VS Code Server` as a service is not allowed
 - you may not provide the software as a stand-alone offering or combine it with any of your applications for others to use, or transfer the software or this agreement to any third party.

So, if I ship code-server [helm][4] charts with my commercial cloud application, will it be breaking VS Code Server's license terms?

  [1]: https://coder.com/docs/code-server/latest
  [2]: https://code.visualstudio.com/docs/remote/vscode-server
  [3]: https://aka.ms/vscode-server-license
  [4]: https://coder.com/docs/code-server/latest/helm

**Comment by code-asher** [comment, marked as answer]
I might have it wrong since I am not a lawyer but from what I understand the difference is between the open source and the final packaged/built VS Code Server product (which, among other things, contains additions not in the open source).  You can host the former, but not the latter.
